repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/files/generate-initram.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC2164 # The directory changes are run as root
```

Comment:
This suppresses the SC2164 warning for the whole file. This warning requires each change of directory to handle its failure and I'm not certain if that is required or if it can be suppressed.

Reply:
Maybe I am wrong, but there are only 2 pairs of `pushd` and `popd`, so it should not be that much code to check their result. It is unlikely that they fail, however, I would rather see a meaningful error message when they do rather than disabling the warning globally.